COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/Point.cs

```diff
@@ -41,6 +47,23 @@ public double DistanceTo(Point point)
 
             return distanceBetwPoints;
         }
+        public double Area()
+        {
+            double area = 0.0;
+            return area;
+        }
+
+        public double Perimeter()
+        {
+            double perimeter = 0.0;
+            return perimeter;
+        }
+
+        public List<Point> GetPoints()
+        {
+            List<Point> points = null;
+            return points;
```

Comment:
why you return null?)) return point itself

Reply:
can i return Point when class type is List of Points?

Reply:
you can return new List<Point> { this } ;

Reply:
or `new List<Point> { new Point(X, Y) }`